pkg/orchestration/wiring/wiringutil/svccatentangler/plugin.go

```diff
@@ -118,8 +130,20 @@ func (e *SvcCatEntangler) WireUp(resource *orch_v1.StateResource, context *wirin
 		return nil, false, err
 	}
 
-	result := &wiringplugin.WiringResult{
-		Resources: []wiringplugin.WiredSmithResource{serviceInstance},
+	var result *wiringplugin.WiringResult
+	if e.OutputResourceContract {
```

Comment:
Why do we need this? Why not always output a contract here?

Reply:
Ok, the issue is that some resources that use `SvcCatEntangler` may not want a contract or want a different one. That is why in my branch, I made it configurable.